commonui/src/main/java/com/hyperwallet/android/ui/common/insight/HyperwalletInsight.java

```diff
@@ -221,12 +221,19 @@ public Handler getHandler() {
     public void trackError(@NonNull final Context context, @NonNull final String pageName,
             @NonNull final String pageGroup, @NonNull final Map<String, String> errorInfoMap) {
 
+        HashMap<String, String> params = new HashMap<>(2);
+        params.put(InsightEventTag.InsightEventTagEventParams.PRODUCT,
+                errorInfoMap.get(InsightEventTag.InsightEventTagEventParams.PRODUCT));
+        params.put(InsightEventTag.InsightEventTagEventParams.PAGE_TECHNOLOGY,
+                errorInfoMap.get(InsightEventTag.InsightEventTagEventParams.PAGE_TECHNOLOGY));
```

Comment:
I'm not a fan of this, but not sure how else to do it. Personally, I think we should get rid of ErrorInfo object in favour of a Map so we can take advantage of loose typing.